### DISCUSSION TITLE
Update React's Peer Dependency

### DISCUSSION THREAD

**Discussion by caioc** [reduxjs/react-redux#2141] [Q&A]
There's any idea when will change react's peer depedency for something like `"react": "^18"`??? I didn't understand why was set to 18.0 :/

**Comment by markerikson** [comment]
I don't understand the question :)

It's currently: `"react": "^18.0"`.

The `^` means "anything equal to or greater than this version".  So, `"^18.0"` means "18.0", "18.1", "18.9999999", etc.
